libs/sdk-ui-kit/src/Dialog/StylingEditorDialog/StylingEditorDialog.tsx

```diff
@@ -6,6 +6,7 @@ import cx from "classnames";
 import { StylingExample } from "./StylingExample";
 import { IColorPalette, ITheme, ObjRef } from "@gooddata/sdk-model";
 import { BubbleHeaderSeparator } from "./BubbleHeaderSeparator";
+import { Button } from "../../Button";
 import { StylingEditorDialogFooter, IStylingEditorDialogFooterProps } from "./StylingEditorDialogFooter";
 import { IntlWrapper } from "@gooddata/sdk-ui";
```

Comment:
Please re-arrange the import section a little bit so that import statements of 3rd-party libraries and components from a SDK package are at the top of the file.

Reply:
Fixed. Please check again. Thanks.